From 032a92532e2b65181f514802ea16f41a7330a92c Mon Sep 17 00:00:00 2001
From: Sergii Pylypenko <x.pelya.x@gmail.com>
Date: Wed, 5 May 2021 22:51:29 +0300
Subject: [PATCH 0674/1053] Emscripten: disabled libtimidity for now, it does
 not work anyway on Chrome for Android because of massive lag

---
 os/emscripten/emscripten-build.sh | 30 ++++++++++++++++--------------
 1 file changed, 16 insertions(+), 14 deletions(-)

diff --git a/os/emscripten/emscripten-build.sh b/os/emscripten/emscripten-build.sh
index b0ca9d63ee..1179564a26 100755
--- a/os/emscripten/emscripten-build.sh
+++ b/os/emscripten/emscripten-build.sh
@@ -107,15 +107,6 @@ autoreconf -V || exit 1
 	cd ..
 }
 
-[ -e build-host ] || {
-	rm -rf build-host
-	mkdir -p build-host
-	cd build-host
-	cmake ../.. -DOPTION_TOOLS_ONLY=ON || exit 1
-	make -j8 tools || exit 1
-	cd ..
-}
-
 mkdir -p baseset
 
 [ -e baseset/opengfx-0.6.1.tar ] || {
@@ -156,16 +147,27 @@ mkdir -p baseset
 #	wget -nc 'https://sourceforge.net/p/mscore/code/HEAD/tree/trunk/mscore/share/sound/TimGM6mb.sf2?format=raw' -O TimGM6mb.sf2 || exit 1
 #}
 
-[ -e timidity/timidity.cfg ] || {
-	wget -nc https://sourceforge.net/projects/libsdl-android/files/timidity.zip || exit 1
-	unzip timidity.zip
+#[ -e timidity/timidity.cfg ] || {
+#	wget -nc https://sourceforge.net/projects/libsdl-android/files/timidity.zip || exit 1
+#	unzip timidity.zip
+#}
+
+[ -e build-host ] || {
+	rm -rf build-host
+	mkdir -p build-host
+	cd build-host
+	cmake ../.. -DOPTION_TOOLS_ONLY=ON || exit 1
+	make -j8 tools || exit 1
+	cd ..
 }
 
 [ -e Makefile ] || emcmake cmake .. -DHOST_BINARY_DIR=$(pwd)/build-host -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DOPTION_USE_ASSERTS=OFF \
-									-DTimidity_LIBRARY=`pwd`/libtimidity-0.2.7/build-wasm/lib/libtimidity.a \
-									-DTimidity_INCLUDE_DIR=`pwd`/libtimidity-0.2.7/build-wasm/include \
 									 || exit 1
 
+#									-DTimidity_LIBRARY=`pwd`/libtimidity-0.2.7/build-wasm/lib/libtimidity.a \
+#									-DTimidity_INCLUDE_DIR=`pwd`/libtimidity-0.2.7/build-wasm/include \
+
+
 [ -z "$NO_CLEAN" ] && rm -f openttd.html
 emmake make -j8 VERBOSE=1 || exit 1
 
-- 
2.55.0

